--- /aarch64-linux-musl-cross/aarch64-linux-musl/include/asm/sigcontext.h
+++ /tmp/aarch64-linux-musl-cross/aarch64-linux-musl/include/asm/sigcontext.h
@@ -25,7 +25,6 @@
  * Signal context structure - contains all info to do with the state
  * before the signal handler was invoked.
  */
-#if false
 struct sigcontext {
 	__u64 fault_address;
 	/* AArch64 registers */
@@ -128,7 +127,6 @@
 	__u16 vl;
 	__u16 __reserved[3];
 };
-#endif
 
 #endif /* !__ASSEMBLY__ */
 
